src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        Ball lotteryBall = lottery.getRandomBall();
+        System.out.println(lotteryBall);
```

Comment:
You need to create three balls using the `getRandomBall()` method and print their information. Consider using a loop to create and print three balls, as recommended in the checklist: 'Use for loop for creating several objects of the same class'.